tuf/pyca_crypto_keys.py

```diff
@@ -404,7 +404,7 @@ def verify_rsa_signature(signature, signature_method, public_key, data):
   tuf.formats.NAME_SCHEMA.check_match(signature_method)
 
   # Does 'signature' have the correct format?
-  tuf.formats.PYCRYPTOSIGNATURE_SCHEMA.check_match(signature)
+  tuf.formats.PYCACRYPTOSIGNATURE_SCHEMA.check_match(signature)
```

Comment:
Following this change, there are two test cases that need to verify that a `tuf.FormatError` exception is raised.

1. https://github.com/theupdateframework/tuf/blob/develop/tests/test_pyca_crypto_keys.py#L83-L84

2. [Line 87](https://github.com/theupdateframework/tuf/blob/develop/tests/test_pycrypto_keys.py#L87) of `test_pycrypto_keys.py` can be changed to:
```Python
self.assertRaises(tuf.FormatError, pycrypto.create_rsa_signature, private_rsa, '')
```